src/components/o-testimonials/index.scss

```diff
@@ -0,0 +1,147 @@
+@import "../../styles/settings/colors";
+@import "../../styles/typo/typo-mixins";
+@import "../../styles/mixins/word-break";
+
+$o-testimonials-padding-vertical-xs: 24px;
+$o-testimonials-padding-vertical-md: 48px;
+$o-testimonials-padding-vertical-lg: $o-testimonials-padding-vertical-md;
```

Comment:
why this variable if there is no difference?

Reply:
naming.. i think it make sense as it is

Reply:
You are right Andy. I will remove this line of code. I don't even need the lg variable.